redis/src/aio/multiplexed_connection.rs

```diff
@@ -146,7 +146,7 @@ where
                 Ok(item) => {
                     if !skip_value {
                         entry.buffer = Some(match entry.buffer.take() {
-                            Some(Value::Array(mut values)) => {
+                            Some(Value::Array(mut values)) if entry.current_response_count > 1 => {
```

Comment:
I'm not familiar with the code and it's really unclear to me what we do here :|
when entry.buffer is Value::Array and entry. current_response_count < 1? what happens if current_response_count == 1? can you add a comment to make it more readable?

Reply:
added documentation.